components/BUILD.gn

```diff
@@ -133,6 +135,11 @@ if (is_ios) {
     ]
     outputs = [ "{{bundle_resources_dir}}/{{source_file_part}}" ]
   }
+  bundle_data("brave_wallet_test_data_bundle") {
+    testonly = true
+    sources = [ "//brave/test/data/brave_wallet/1.0.1/images/logo.png" ]
+    outputs = [ "{{bundle_resources_dir}}/brave/test/data/brave_wallet/1.0.1/images/{{source_file_part}}" ]
+  }
```

Comment:
It doesn't feel right if we have to enumerate test data files in bundle deps. Should be something more generic. 
@bridiver could you please suggest something?

Reply:
Hmm... I was sure I fixed some of these unit tests on ios by adding the files to `data = [ ... ]`, but maybe that only worked for android. It's still the case either way that `data = [ "//brave/components/test/data/brave_wallet" ]` should be added to the relevant test target as that is needed for rebuilds as explained above